docs/First-Installation.md

```diff
@@ -58,44 +58,40 @@ The simplest way to get most of these components is through an all-in-one kit su
+6. Plug in a controller or keyboard
+7. Turn on the Raspberry Pi
 
-## Configure Controllers
+## First Boot
 
-On first boot your filesystem will be expanded automatically, you will then be welcomed with the following screen- this menu will configure your controls for both EmulationStation and RetroArch Emulators:
+On first boot you will be welcomed with the screen below. This menu will configure your controls for both EmulationStation and all RetroArch emulators:
 
 ![welcomescreen](https://cloud.githubusercontent.com/assets/10035308/9140482/cf42f25c-3cee-11e5-8f91-c1fc1c57175c.png)
 
-Hold down any button on your keyboard or gamepad and the name will appear at the bottom and then open up into a configuration menu:
+Hold down any button on your controller or keyboard. While holding, its name will appear at the bottom of the window for a few seconds and then open up into a configuration menu:
 
-![welcomescreengamepadname](https://cloud.githubusercontent.com/assets/10035308/9140505/f5c19e38-3cee-11e5-965e-0e4e85ddaf56.png)
+![welcomescreengamepadconfigure](https://cloud.githubusercontent.com/assets/10035308/9140518/0263b9c8-3cef-11e5-922f-42f790f3be91.png)
 
-Follow the onscreen instructions to configure your gamepad- if you run out of buttons just hold down a button to skip each unused button. **When you get to OK press the button you have configured as "A"**.
+Follow the onscreen instructions to configure your gamepad. If your controller doesn't have a button that you're being asked to define, just hold down any button to skip it.
 
-![welcomescreengamepadconfigure](https://cloud.githubusercontent.com/assets/10035308/9140518/0263b9c8-3cef-11e5-922f-42f790f3be91.png)
+The [Hotkey](First-Installation#Hotkeys) button is explained in detail below and can be changed later, but for now it is suggested to be defined as the **Select** button.
 
-If you wish to configure more than one controller, you can do so from the start menu of EmulationStation. For more details on manual controller configurations see this page [Here](RetroArch-Configuration).
+**When you get to "OK" at the end, press the button you have configured as "A" (East) to complete this step**.
 
 See the following diagrams for reference:
 
-| SNES Controller | 
-|:---:|
-|![snes_controller](https://cloud.githubusercontent.com/assets/10035308/22185414/f129dc28-e099-11e6-8524-93facf275eda.png)|
-
 | XBox 360 Controller |
 |:---:|
 |![xbox360_controller](https://cloud.githubusercontent.com/assets/10035308/22185415/f12ff342-e099-11e6-8adb-d18e9c638e94.png)|
 
-| PS3 Controller |
-|:---:|
-|![ps3_controller](https://cloud.githubusercontent.com/assets/10035308/22185413/f10f27de-e099-11e6-97a4-ecbbc82c9e46.png)|
+Also see the diagrams for [SNES Controller](https://cloud.githubusercontent.com/assets/10035308/22185414/f129dc28-e099-11e6-8524-93facf275eda.png) and [PS3 Controller](https://cloud.githubusercontent.com/assets/10035308/22185413/f10f27de-e099-11e6-97a4-ecbbc82c9e46.png)
```

Comment:
no please lease the common layouts in here.

Reply:
I don't understand. Do you want me to keep the large image embeds instead of turning them into links?

Reply:
Yes, I think they should stay as embedded images. Linking images seems daft